coderadar-vcs/src/main/java/io/reflectoring/coderadar/vcs/adapter/GetRawCommitContentAdapter.java

```diff
@@ -56,7 +56,13 @@
       } else {
         rt1 = new RawText(new byte[0]);
       }
-      RawText rt2 = new RawText(BlobUtils.getRawContent(git.getRepository(), commit, filepath));
+      byte[] rawContent = BlobUtils.getRawContent(git.getRepository(), commit, filepath);
+      RawText rt2;
+      if (rawContent != null && rawContent.length != 0) {
+        rt2 = new RawText(BlobUtils.getRawContent(git.getRepository(), commit, filepath));
+      } else {
+        rt2 = new RawText(new byte[] {'\n'});
```

Comment:
An anderer Stelle wird `%n` verwendet, was ein Plattformunabhängiger Zeilenumbruch ist. Sollte der hier auch verwendet werden?

Reply:
Wo wird `%n` verwendet?

Reply:
https://github.com/adessoAG/coderadar/blob/b36d3c17aac94e8bf2c603a912e41a1fbfcbb158/coderadar-core/src/main/java/io/reflectoring/coderadar/analyzer/service/AnalyzerPluginService.java#L106

Reply:
I don't think this will change anything in the given use case.